content/engineering/languages-runtimes/javascript.md

```diff
@@ -234,8 +234,7 @@ editor and build system, as described at
 [http://eslint.org/docs/user-guide/integrations](http://eslint.org/docs/user-guide/integrations).
 Using an eslint plugin with your editor makes it easier to quickly see
 non-conforming lines of code on the spot, as shown below:
-
-![Atom eslint plugin screenshot]({{ site.baseurl }}/javascript/atom-eslint-example.png)
+{% image_with_class "assets/engineering/img/javascript/atom-eslint-example.png"  "Atom eslint plugin screenshot" %}
```

Comment:
`image _with_class` takes 3 arguments, `<file>` `<class>` `<alt-text>`, so this is the assigning the image classes "Atom eslint plugin screenshot" and assigning the alt text as the literal word "undefined" (not just blank). All to say, you want to add an empty `"" `between the file name and the alt text here.